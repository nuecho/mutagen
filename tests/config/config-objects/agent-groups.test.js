const checkKey = (cfgObject) => {return cfgObject.group.name === "cfgObjectTest"};

cfgObjectTests("agentGroups", "config-objects/agent-groups-config.json", checkKey);
