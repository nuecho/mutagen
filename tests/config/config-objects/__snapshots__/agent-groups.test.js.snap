// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mutagen agentGroups config import-export] should create the agentGroups: the agentGroups have been created 1`] = `
Object {
  "stderr": "",
  "stdout": "+ agentgroup => Environment/cfgObjectTest
Completed. 1 object(s) imported.
",
}
`;

exports[`[mutagen agentGroups config import-export] should update the agentGroups: the agentGroups have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ agentgroup => Environment/cfgObjectTest
Completed. 1 object(s) imported.
",
}
`;
